quadcontrol/config.h

```diff
@@ -34,4 +35,10 @@ extern int config_throttleRead(const char *path, quad_throttle_t **throttle, int
 extern int config_attitudeRead(const char *path, quad_att_t **attitude, int *sz);
 
 
+/*
+Parses no more than first attenuation config structures from file defined by `path`. If succeeded returns 0 and `atten` is array with length equal to one.
+Returns -1 on error ir multiple attenuation structures.
```

Comment:
Typo - `ir` should be `or`
```suggestion
Returns -1 on error or multiple attenuation structures.
```